modal_popup.js

```diff
@@ -0,0 +1,142 @@
+    buttons: 'See project',
+    description:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+  },
+
+  {
+    image: 'images/project_image.svg',
+    title: ['Multi-Post Stories   Gain+Glory', ' Keeping track of hundreds of components'],
+    language: ['Ruby on rails', 'Css', 'JavaScript', 'html'],
+    buttons: 'See project',
+    description:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+  },
+];
+
+const recentWorks = document.querySelector('.grid-container');
+
+recentWorks.innerHTML += myWorks
+  .map(
+    (project) => `
+      <div class="card-flex">
+        <div class="project-image" src='${project.image}' ></div>
+        <div class="description">
+          <h2 class="new">${project.title[0]}</h2>
+          <ul class="list p-lang">
+          ${project.language
+    .map(
+      (technology) => `<li>
+              <button class="btn-btn" type="">${technology}</button>
+            </li>
+          `,
+    )
+    .join('')}
+          </ul>
+          <div class="sec2-btn">
+            <button class="screen-xl btn view" type="button">
+              ${project.buttons}
+            </button>
+          </div>
+    </div>
+  </div>
+  `,
+  )
+  .join('');
+
+// modalPopup
+const modal = document.querySelector('#modal');
+const viewProj = document.querySelectorAll('.btn');
+const popup = myWorks.map(
+  (modalPopup) => `
+  <div id="overlay">
+          <div class="modal-card">
+            <div class="collapse-modal">
+              <a class="default"><i class="fa-sharp fa-solid fa-xmark"></i></a>
+            </div>
+            <div class="modal-img" src='${modalPopup.image
+}' alt=" thumbnail-image">
+            </div>
+            <div class="modal-body">
+              <div class="modal-content">
+                <div class="modal-header">
+                  <h2 class="modal-title">${modalPopup.title[1]}</h2>
+                  <div class="tech-badges">
+                    <ul class="list p-lang mod-badge">
+                    ${modalPopup.language
+    .map(
+      (technology) => `
+                    <li class="btn-btn"> ${technology}</li>
```

Comment:
- This part of your project has some little difference from what is in the Figma design template. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your project:
<img width="354" alt="stjsdykxr" src="https://user-images.githubusercontent.com/73607512/202428959-e7e7d35d-9874-4daa-be8e-ff4082ced2b1.PNG">

Design Template:
<img width="218" alt="tjetet" src="https://user-images.githubusercontent.com/73607512/202429104-8a6adea3-d04f-49fe-ab02-f49f49299ddc.PNG">